tests/unittest_brain_pathlib.py

```diff
@@ -72,7 +77,7 @@ class A:
         parents = 42
 
     c = A()
-    error = c.parents[2]
+    error = c.parents[:2]
```

Comment:
Did this have to be changed?

Reply:
Originally I hadn't fixed the normal index look up, so this was not really testing anything:
The brain returned `Uninferable` and the non-brain would return `Uninferable`. So we weren't actually testing if this doesn't actually enter the brain. By using the slice we can see that this doesn't trigger the brain as it would give a different result for `PY3.10+`.

Now that I fixed the brain to work with index as well the change is perhaps not necessary anymore.